reviews/swagger.yaml

```diff
@@ -0,0 +1,679 @@
+        type: string
+        format: date-time
+        readOnly: true
+      invoice_url:
+        type: string
+        description: The url to send to the customer so that they can complete the checkout.  When using send_invoice, this url is emailed to the customer. This field can be used so that an API client can use another method of communication to provide the url to the customer.
+        readOnly: true
+
+      line_items:
+        type: array
+        items:
+          $ref: '#/definitions/line_item'
+
+      metafields:
+        type: array
+        items:
+          $ref: '#/definitions/metafield'
+
+      shipping_line:
+        type: object
+        items:
+          $ref: '#/definitions/shipping_line'
+
+      tags:
+        type: string
+        description: Tags are additional short descriptors, commonly used for filtering and searching, formatted as a string of comma-separated values. Each individual tag is limited to 40 characters in length.
+      tax_exempt:
+        type: boolean
+        description: Sets whether taxes are exempt for this draft order. If this value is false, Shopify will honor the tax_exempt value for each line_item.
+      tax_lines:
+        type: array
+        items:
+          $ref: '#/definitions/tax_line'
+        readOnly: true
+      discount:
+        type: object
+        items:
+          $ref: '#/definitions/discount'
+      taxes_included:
+        type: boolean
+        description: Shop settings taxes are included in the price
+        readOnly: true
+
+      total_tax:
+        type: string
+        description: Total tax amount
+        readOnly: true
+
+      subtotal_price:
+        type: string
+        description: Subtotal after discounts, before shipping and taxes have been applied
+        readOnly: true
+
+      total_price:
+        type: string
+        description: Includes discounts, shipping and taxes
+        readOnly: true
+
+      completed_at:
+        type: string
+        format: date-time
+        description: Date at which an order was created and the draft order moved to “completed” status.
+        readOnly: true
+      created_at:
+        type: string
+        description: Date at which an order was created
+        readOnly: true
+
+      updated_at:
+        type: string
+        description: Date at which an order was updated
+        readOnly: true
+
+      status:
+        type: string
+        enum:
+        - open
+        - invoice_sent
+        - completed
+        description: String describing the state of the draft order - “open”, “invoice sent”, “completed”.
+        readOnly: true
+
+  draft_order_invoice:
+    type: object
+    properties:
+      to:
+        type: string
+        description: >-
+          Use parameter value, but default to draft order email or customer
+          value if parameter not set. If email is blank everywhere, error out.
+      from:
+        type: string
+        description: One of the staff accoun e-mail addresses.
```

Comment:
This email address that the email will appear to be sent from. This defaults to the shop's customer email, and if none exists, simply the shop's account email. When provided, this email address must correspond to the email address of one of the staff accounts.